src/app/feed/info-box/info-box.component.ts

```diff
@@ -34,6 +35,10 @@ export class InfoBoxComponent implements OnChanges {
 		scaleShowVerticalLines: false,
 		responsive: true
 	};
+	public loklakServerUrl = defaultUrlConfig['loklak']['official'];
```

Comment:
Any reason behind not going with `defaultUrlConfig.loklak.official`?

Reply:
Please avoid doing that. It makes the code less readable.

Reply:
Thanks for helping, I will try to use in this way now on wards, I am making the changes.